app/helpers/decidim/vocdoni/admin/steps_wizard_helper.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Vocdoni
+    module Admin
+      # Custom helpers, scoped to the vocdoni admin engine.
+      #
+      module StepsWizardHelper
+        def tab_class(tab_name, active_class)
+          css_classes = ["tabs-title"]
+          css_classes << "is-active" if active_class == tab_name
+          css_classes.join(" ")
+        end
+
+        def tab_link_class(tab_name, election)
+          case tab_name
+          when "questions"
+            election.present? ? "" : "disabled"
+          when "census"
+            census_ready_or_minimum_answers?(election) ? "" : "disabled"
+          when "results"
+            census_ready? ? "" : "disabled"
+          when "publish"
+            publishable?(election) ? "" : "disabled"
```

Comment:
As we have some duplicated logic with the permission class, and as these two features (the permission check and the tab_link_class helper) are really two sides of the same coin, how about if we extract these methods to the Election model? What I have in mind is something like this: 
```suggestion
        def tab_link_class(tab_name, election)
          case tab_name
          when "questions"
            election.election.ready_for_questions_form? ? "" : "disabled"
          when "census"
            election.ready_for_census_form?  ? "" : "disabled"
          when "results"
            election.ready_for_results_form?  ? "" : "disabled"
          when "publish"
            election.ready_for_publish_form?  ? "" : "disabled"
```

And then in `app/models/decidim/vocdoni/election.rb`:


```ruby

def ready_for_questions_form?
  present?
end 

def ready_for_census_form?
  ...
end 
``` 

By reusing the logic on both of these places we only need to change it on one place in the future when/if the conditions need to change, and it's also easier to test and read.